projects/packages/account-protection/src/class-rest-controller.php

```diff
@@ -0,0 +1,144 @@
+<?php
+/**
+ * Class use to register REST API endpoints used by Account Protection
+ *
+ * @package automattic/jetpack-account-protection
+ */
+
+namespace Automattic\Jetpack\Account_Protection;
+
+use Automattic\Jetpack\Connection\REST_Connector;
+use WP_Error;
+use WP_REST_Request;
+use WP_REST_Response;
+use WP_REST_Server;
+
+/**
+ * Defines our endponts.
+ */
+class REST_Controller {
+	/**
+	 * Register REST API endpoints.
+	 *
+	 * @return void
+	 */
+	public static function register_rest_routes() {
+		// Ensure routes are only initialized once.
+		static $routes_registered = false;
+		if ( $routes_registered ) {
+			return;
+		}
+
+		register_rest_route(
+			'jetpack/v4',
+			'/toggle-account-protection',
+			array(
+				'methods'             => WP_REST_Server::EDITABLE,
+				'callback'            => __CLASS__ . '::api_toggle_account_protection',
+				'permission_callback' => __CLASS__ . '::account_protection_permissions_callback',
+			)
+		);
```

Comment:
The `POST /account-protection` endpoint could be able to make this toggle endpoint redundant, though I haven't gotten that far yet to see if that is currently the case.